components/kyma-environment-broker/internal/process/deprovisioning/initialisation_test.go

```diff
@@ -60,6 +60,43 @@ func TestInitialisationStep_Run(t *testing.T) {
 	assert.Equal(t, domain.Succeeded, operation.State)
 }
 
+func TestInitialisationStep_Run_ShouldDeleteInstanceWhenRuntimeNotExist(t *testing.T) {
```

Comment:
move test to `TestInitialisationStep_Run` and use t.Run() schema:
```go
func TestInitialisationStep_Run(t *testing.T) {
   t.Run("description test", func(t *testing.T) {

  })

  t.Run("description second test", func(t *testing.T) {

  })
}
```

the same for `remove_runtime_test.go`